assets/css/buttons.css

```diff
@@ -14,13 +14,13 @@
   font-weight: 400;
   font-style: normal;
   line-height: 1;
-  font-size: .75rem;
+  font-size: .75em;
```

Comment:
I've been doing a bunch of reading into font sizing the past couple of days, and people on the internet are really conflicted and opinionated. My initial reason I made it em (or rather rem) is because of our intended audience. I didn't know if you guys thought that the font size should be larger. If you wanted it to be larger, then I would just change the font size for the html tag in one place and everything would change accordingly. I don't have much of an opinion either way, so if you think just using pixels makes more sense, then I could easily just change it over to that. Just let me know what you think.